language/Russian/strings.po

```diff
@@ -2339,40 +2370,40 @@ msgstr "Насыщенность луча"
 #: src/frontend/mame/ui/ui.cpp:1502
 #, c-format
 msgid "Crosshair Scale %1$s"
-msgstr ""
+msgstr "Масштаб прицела %1$s"
 
 #: src/frontend/mame/ui/ui.cpp:1502 src/frontend/mame/ui/ui.cpp:1504
 msgid "X"
-msgstr ""
+msgstr "X"
 
 #: src/frontend/mame/ui/ui.cpp:1502 src/frontend/mame/ui/ui.cpp:1504
 msgid "Y"
-msgstr ""
+msgstr "Y"
 
 #: src/frontend/mame/ui/ui.cpp:1504
 #, c-format
 msgid "Crosshair Offset %1$s"
-msgstr ""
+msgstr "Смещение прицела %1$s"
 
 #: src/frontend/mame/ui/ui.cpp:1592
 #, c-format
 msgid "%1$3ddB"
-msgstr ""
+msgstr "%1$3dдБ"
```

Comment:
Does Russian use transliterated "дБ" for "decibels", but untransliterated "fps" for "frames per second"?

Reply:
дБ is correct abbrev of term "децибел", but there is no abbrev for fps = "кадров в секунду" in Russian language, sometimes its called "фпс" but its more like slang, not sure if at acceptable in our project.

in general, I found this submission not acceptable "as is", number of terms translated to very specific slang or self-made transliterations, not to reall Russian language, so it will not be understood by regular users/people. other parts (later strings mostly) looks like automatically translated by poedit or similar software, and have wrong context meaning.
but here is also number of good translated strings or corrections. I'll try to filter good/bad parts during coming days, before this dev cycle freeze.

Reply:
Freeze is today, but I'll cherry-pick translation updates before release if necessary.

Reply:
@cuavas done. I'm not 100% happy with it but it is good milestone.